cyphermain.py

```diff
@@ -118,6 +132,7 @@ def single_arg_encrypt_file(in_filename):
     encrypt_file(key, in_filename)
 
 def select_files():
+    global encrypted_count

```

Comment:
Please use method arguments instead of global variables when not returning in a particular method.